src/Components/ArticleList.jsx

```diff
@@ -29,25 +31,27 @@ export default function ArticleList() {
   return (
     <div className="articles-container">
       {articles.map((article) => (
-        <article key={article.article_id} className="article-card">
-          <img
-            src={article.article_img_url}
-            alt={article.title}
-            className="article-image"
-          />
-          <div className="article-content">
-            <h2>{article.title}</h2>
-            <p className="article-meta">
-              By {article.author} •{" "}
-              {new Date(article.created_at).toLocaleDateString()}
-            </p>
-            <p className="article-topic">Topic: {article.topic}</p>
-            <div className="article-stats">
-              <span> {article.comment_count} comments</span>
-              <span> {article.votes} votes</span>
+        <Link to={`/articles/${article.article_id}`} key={article.article_id}>
+          <article className="article-card">
+            <img
+              src={article.article_img_url}
+              alt={article.title}
+              className="article-image"
+            />
+            <div className="article-content">
+              <h2>{article.title}</h2>
+              <p className="article-meta">
+                By {article.author} •{" "}
+                {new Date(article.created_at).toLocaleDateString()}
+              </p>
+              <p className="article-topic">Topic: {article.topic}</p>
+              <div className="article-stats">
+                <span> {article.comment_count} comments</span>
+                <span> {article.votes} votes</span>
+              </div>
```

Comment:
It would be nicer to abstract this part into another ArticleCard component to make it reusable.